edgelet/iotedge/src/check/mod.rs

```diff
@@ -45,6 +45,7 @@ pub struct Check {
 
     // These optional fields are populated by the checks
     iothub_hostname: Option<String>, // populated by `aziot check`
+    proxy_uri: Option<String>,       // populated by `aziot check`
```

Comment:
Should this be a Uri type, such as `hyper::Uri`?

Reply:
The string is passed verbatim to aziotctl and never used in iotedge. So, the type could be serialized upfront here.